test/gateway/eip20_cogateway/confirm_stake_intent.js

```diff
@@ -356,7 +355,7 @@ contract('EIP20CoGateway.confirmStakeIntent() ', function (accounts) {
         "Merkle proof verification failed.",
       );
 
-  });
+    });
```

Comment:
The file doesn't have consistent spacing, either 2 or 4 not mix.

Reply:
Thanks, fixed 👍